### PR TITLE
Pioneer 14b Glider Deploy csv Updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Deployments:
376:Vehicle Name: cp_376
Curr Time: Wed Aug 19 17:11:08 2020 MT: 1065
GPS Location: 3953.111 N -7023.747 E measured 254.094 secs ago
379:
Curr Time: Wed Aug 19 17:18:10 2020 MT: 879
GPS Location: 3953.109 N -7023.815 E measured 68.821 secs ago
583: Vehicle Name: cp_583
Curr Time: Wed Aug 19 19:25:46 2020 MT: 1298
GPS Location: 4002.282 N -7041.896 E measured 64.65 secs ago

Recoveries:
388: Wed Aug 19 22:00:20 2020
GPS Location:  3949.571 N -7110.479 E measured     73.078 secs ago
*514:*Wed Aug 19 21:26:03 2020
GPS Location:  3949.454 N -7104.770 E

**Note: We still need to add deployment cruise TS20200819 to CruiseInformation csv

### CODE REVIEW

File: deployment/CP05MOAS-PG583_Deploy.csv [hunk 1]
```diff
@@ -12,4 +12,11 @@ TS20200613,,MZ20200731,,CP05MOAS-PG583-02-DOSTAM000,2,1,2020-06-13T20:10:46,2020
...
+TS20200819,,,,CP05MOAS-PG583-00-ENG000000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGCON-GLDRCP-00583,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,,,CP05MOAS-PG583-01-CTDGVM000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-CTDGVM-09307,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,,,CP05MOAS-PG583-02-DOSTAM000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-DOSTAM-00539,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,,,CP05MOAS-PG583-03-FLORTM000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTM-04093,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,,,CP05MOAS-PG583-04-FLORTO000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTO-01348,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,,,CP05MOAS-PG583-05-NUTNRM000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-NUTNRM-00709,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,,,CP05MOAS-PG583-06-PARADM000,3,1,2020-08-19T19:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-PARADM-50218,40.03803,-70.69827,,200,200,CNPM
```
Comment (snwhite) [inline deployment/CP05MOAS-PG583_Deploy.csv:22]: Times look good.
Instrument S/Ns compared to FCP 3304-00180-00005

File: deployment/CP05MOAS-GL514_Deploy.csv [hunk 2]
```diff
@@ -5,9 +5,9 @@ TS20200301,,TS20200522,,CP05MOAS-GL514-02-FLORTM000,1,1,2020-03-02T09:58:39,2020
...
\ No newline at end of file
+TS20200613,,TS20200819,,CP05MOAS-GL514-00-ENG000000,2,1,2020-06-13T21:30:56,2020-08-19T21:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00514,CGCON-GLDRCP-00514,39.93223,-70.36995,,1000,1000,FZ-1
+TS20200613,,TS20200819,,CP05MOAS-GL514-01-ADCPAM000,2,1,2020-06-13T21:30:56,2020-08-19T21:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00514,CGINS-ADCPAM-50867,39.93223,-70.36995,,1000,1000,FZ-1
+TS20200613,,TS20200819,,CP05MOAS-GL514-02-FLORTM000,2,1,2020-06-13T21:30:56,2020-08-19T21:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00514,CGINS-FLORTM-03207,39.93223,-70.36995,,1000,1000,FZ-1
+TS20200613,,TS20200819,,CP05MOAS-GL514-03-CTDGVM000,2,1,2020-06-13T21:30:56,2020-08-19T21:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00514,CGINS-CTDGVM-09362,39.93223,-70.36995,,1000,1000,FZ-1
+TS20200613,,TS20200819,,CP05MOAS-GL514-04-DOSTAM000,2,1,2020-06-13T21:30:56,2020-08-19T21:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00514,CGINS-DOSTAM-00604,39.93223,-70.36995,,1000,1000,FZ-1
+TS20200613,,TS20200819,,CP05MOAS-GL514-05-PARADM000,2,1,2020-06-13T21:30:56,2020-08-19T21:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00514,CGINS-PARADM-50204,39.93223,-70.36995,,1000,1000,FZ-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL514_Deploy.csv:13]: Times look good.

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 3]
```diff
@@ -53,9 +53,9 @@ TS20191212,,TS20200301,,CP05MOAS-GL388-02-FLORTM000,9,1,2019-12-12T22:47:17,2020
...
\ No newline at end of file
+TS20200613,,TS20200819,,CP05MOAS-GL388-00-ENG000000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.92978,-70.36552,,1000,1000,SS-2
+TS20200613,,TS20200819,,CP05MOAS-GL388-01-ADCPAM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-49016,39.92978,-70.36552,,1000,1000,SS-2
+TS20200613,,TS20200819,,CP05MOAS-GL388-02-FLORTM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03185,39.92978,-70.36552,,1000,1000,SS-2
+TS20200613,,TS20200819,,CP05MOAS-GL388-03-CTDGVM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-00073,39.92978,-70.36552,,1000,1000,SS-2
+TS20200613,,TS20200819,,CP05MOAS-GL388-04-DOSTAM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00323,39.92978,-70.36552,,1000,1000,SS-2
+TS20200613,,TS20200819,,CP05MOAS-GL388-05-PARADM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50244,39.92978,-70.36552,,1000,1000,SS-2
```
Comment (snwhite) [inline deployment/CP05MOAS-GL388_Deploy.csv:61]: Times look good.

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 4]
```diff
@@ -52,4 +52,10 @@ TS20200301,,TS20200522,,CP05MOAS-GL379-01-ADCPAM000,9,1,2020-03-02T09:44:16,2020
...
\ No newline at end of file
+TS20200301,,TS20200522,,CP05MOAS-GL379-05-PARADM000,9,1,2020-03-02T09:44:16,2020-05-22T17:09:47,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,39.9163,-70.36053,,1000,1000,SS-1
+TS20200819,,,,CP05MOAS-GL379-00-ENG000000,10,1,2020-08-19T17:18:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,,,CP05MOAS-GL379-01-ADCPAM000,10,1,2020-08-19T17:18:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-50865,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,,,CP05MOAS-GL379-02-FLORTM000,10,1,2020-08-19T17:18:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,,,CP05MOAS-GL379-03-CTDGVM000,10,1,2020-08-19T17:18:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,,,CP05MOAS-GL379-04-DOSTAM000,10,1,2020-08-19T17:18:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00171,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,,,CP05MOAS-GL379-05-PARADM000,10,1,2020-08-19T17:18:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.88515,-70.39692,,1000,1000,FZ-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL379_Deploy.csv:61]: Times look good.
Instrument S/Ns compared to FCP 3304-00150-00124

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 5]
```diff
@@ -64,4 +64,10 @@ TS20200301,,TS20200522,,CP05MOAS-GL376-01-ADCPAM000,11,1,2020-03-02T09:46:04,202
...
\ No newline at end of file
+TS20200301,,TS20200522,,CP05MOAS-GL376-05-PARADM000,11,1,2020-03-02T09:46:04,2020-05-22T18:00:27,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.91503,-70.361067,,1000,1000,SS-2
+TS20200819,,,,CP05MOAS-GL376-00-ENG000000,12,1,2020-08-19T17:11:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,,,CP05MOAS-GL376-01-ADCPAM000,12,1,2020-08-19T17:11:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,,,CP05MOAS-GL376-02-FLORTM000,12,1,2020-08-19T17:11:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03116,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,,,CP05MOAS-GL376-03-CTDGVM000,12,1,2020-08-19T17:11:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09089,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,,,CP05MOAS-GL376-04-DOSTAM000,12,1,2020-08-19T17:11:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00543,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,,,CP05MOAS-GL376-05-PARADM000,12,1,2020-08-19T17:11:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.88518,-70.39578,,1000,1000,SS-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL376_Deploy.csv:73]: Times look good.
Instrument S/Ns compared to FCP 3304-00150-00125

Comment (asmith-whoi) [inline deployment/CP05MOAS-GL376_Deploy.csv:73]: OK

File: deployment/CP05MOAS-GL388_Deploy.csv (hunk 3 above)
Comment (asmith-whoi) [inline deployment/CP05MOAS-GL388_Deploy.csv:61]: OK

File: deployment/CP05MOAS-GL514_Deploy.csv (hunk 2 above)
Comment (asmith-whoi) [inline deployment/CP05MOAS-GL514_Deploy.csv:13]: OK

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 4 above)
Comment (asmith-whoi) [inline deployment/CP05MOAS-GL379_Deploy.csv:61]: OK

File: deployment/CP05MOAS-PG583_Deploy.csv (hunk 1 above)
Comment (asmith-whoi) [inline deployment/CP05MOAS-PG583_Deploy.csv:22]: OK

**Pull request merged**